src/test/java/com/karankumar/bookproject/backend/service/CustomShelfServiceTest.java

```diff
@@ -0,0 +1,67 @@
+package com.karankumar.bookproject.backend.service;
```

Comment:
Could you please add our [copyright notice](https://github.com/Project-Books/book-project/blob/master/docs/COPYRIGHT) to the top of this file?